finance/angular/src/app/finance-example/finance-example.component.ts

```diff
@@ -66,6 +82,7 @@ export class FinanceExample {
   @Input() isDarkMode: boolean = false;
 
   themeClass = `${this.gridTheme}${this.isDarkMode ? '-dark' : ''}`;
+  theme: GridOptions['theme'] = 'legacy';
```

Comment:
We should look to upgrade to the new theme as soon as possible

Reply:
Definitely 👍🏼